src/pages/BalanceNew/StuckOraib/useGetOraiBridgeBalances.tsx

```diff
@@ -21,14 +22,18 @@ export default function useGetOraiBridgeBalances(moveOraib2OraiLoading: boolean)
             }
             const client = await StargateClient.connect(ORAI_BRIDGE_RPC);
             const data: readonly Coin[] = await client.getAllBalances(oraiBridgeAddress);
-            const remainingOraib = data.filter((item: Coin) => toDisplay(item.amount, ORAIB_DECIMALS) > 0)
-                .map((item: Coin) => {
```

Comment:
I think this file can have several test cases to test filtering & querying OraiBridge balances